openmdao/core/system.py

```diff
@@ -715,6 +717,612 @@ def _declare_options(self):
         """
         pass
 
+    def _have_output_solver_options_been_applied(self):
+        """
+        Check to see if the cached output solver options were applied.
+        """
+        been_applied = True
+        for subsys in [self] + self._subsystems_myproc:
```

Comment:
So is the intent that this only checks one level down and doesn't recurse?  Also, could use `for subsys in self.system_iter(include_self=True, recurse=False)` here (if no recursion is the intent)

Reply:
Thanks for the tip. Used your advice.